jupyterhub/templates/scheduling/user-scheduler/_helpers.tpl

```diff
@@ -0,0 +1,12 @@
+{{- /*
+Renders the kube-scheduler's image based on .Values.scheduling.userScheduler.name and
+optionally on .Values.scheduling.userScheduler.tag. The default tag is set to the clusters
+kubernetes version.
+*/}}
+{{- define "jupyterhub.scheduler.image" -}}
+{{- $name := .Values.scheduling.userScheduler.image.name -}}
+{{- $valuesVersion := .Values.scheduling.userScheduler.image.tag -}}
+{{- $clusterVersion := (split "-" .Capabilities.KubeVersion.GitVersion)._0 -}}
+{{- $tag := $valuesVersion | default $clusterVersion -}}
+{{ $name }}:{{ $tag }}
+{{- end }}
```

Comment:
**This is currently not in use.**
- [ ] Make a note about this not being in use yet, but could be if we want to match the k8s version.